utils/utils.py

```diff
@@ -67,15 +68,32 @@ def flatten_dict_observations(env):
     return gym.wrappers.FlattenDictWrapper(env, dict_keys=list(keys))
 
 
-def make_env(env_id, rank=0, seed=0, log_dir=None):
+def treat_env_wrapper_hyperparams(hyperparams):
+    def get_module_name(fullname):
+        return '.'.join(wrapper_name.split('.')[:-1])
+    
+    def get_class_name(fullname):
+        return wrapper_name.split('.')[-1]
+    
+    if 'env_wrapper' in hyperparams.keys():
+        wrapper_name = hyperparams.get('env_wrapper')
+        wrapper_module = importlib.import_module(get_module_name(wrapper_name))
+        return getattr(wrapper_module, get_class_name(wrapper_name))
+    else:
+        return None
+
+
+def make_env(env_id, rank=0, seed=0, log_dir=None, wrapper_class=None):
     """
     Helper function to multiprocess training
     and log the progress.
-
+    
     :param env_id: (str)
     :param rank: (int)
     :param seed: (int)
     :param log_dir: (str)
+    :param wrapper: a subclass of gym.Wrapper to wrap the original env
```

Comment:
```suggestion
    :param wrapper: (gym.Wrapper) a subclass of gym.Wrapper to wrap the original env
```

Reply:
It is a type object for a subclass of gym.Wrapper.
In a generic class representation, it could be something like type<gym.Wrapper>, which is not common in Python.

Reply:
ah I see, I read the code too quickly, I thought it was an instance of a subclass, in fact, it is only a subclass of `gym.Wrapper`.